src/source/Metrics/Metrics.h

```diff
@@ -72,11 +72,12 @@ STATUS getRtpRemoteInboundStats(PRtcPeerConnection, PRtcRemoteInboundRtpStreamSt
 /**
  * @brief Get RTP outbound stats
  * @param [in] PRtcPeerConnection
+ * @param [in] PRtcRtpTransceiver transceiver to get stats for, first transceiver if NULL
  * @param [in/out] PRtcOutboundRtpStreamStats Fill up the RTP outbound stats for application consumption
  * @return Pass/Fail
  *
  */
-STATUS getRtpOutboundStats(PRtcPeerConnection, PRtcOutboundRtpStreamStats);
+STATUS getRtpOutboundStats(PRtcPeerConnection, PRtcRtpTransceiver, PRtcOutboundRtpStreamStats);
```

Comment:
@disa6302 like so

Reply:
Great! Thanks @zhuker !

One question, just to confirm, if the application requests stats for a specific transceiver, it will have to call `getRtpOutboundStats` directly right? Because, with the current `rtcPeerConnectionGetMetrics`, that is not possible. I will be modifying the public API to accomodate these cases later, but we can expose `getRtpOutboundStats` in public header so that it is accessible to pass in other parameters is what I am thinking. What do you think?